### PR TITLE
add html str. and style for ahmad cv

### PR DESCRIPTION
**Pull request by ghassanaldarwish**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            <li>Development of a company's telecommunications policies</li>
+          </ul>
+        </section>
+
+      </section>
+
+      <!-- ///////////////////////////////////-->
+      <section>
+        <h2>EDUCATIONAL BACKGROUND:</h2>
+        <p>2010-2012</p>
```
Comment (mihneavlad) [inline index.html:99]: Add markup for time (year or date) whenever you have this information.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        <dl>
+          <dt><strong>High School Graduation</strong></dt>
+          <dd>Night School Kunduz Afghanistan
+          </dd>
+          <dd>Higher Secondary School Mawlawi Sarajuddin Shaheed High School Kunduz Afghanistan (2009-2010)</dd>
```
Comment (mihneavlad) [inline index.html:104]: Add markup for time (year or date) whenever you have this information.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+          <dd>High Secondary School in Sadramazdan High School Nikpai Village Kunduz Afghanistan (2007-2009)</dd>
```
Comment (mihneavlad) [inline index.html:105]: Add markup for time (year or date) whenever you have this information.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    </header>
+
+
+    <main>
+
+
+
+      <section>
+        <h2>PERSONAL INFORMATION</h2>
+        <p><strong>first name</strong> : Ahmad <br>
```
Comment (mihneavlad) [inline index.html:24]: Change to capital letters, First Name to each field below h2

**Review by ghassanaldarwish — COMMENTED** [review]
change the time

**Pull request merged**